docs: sync visualization docs with source (pt3) (#126814)
grafana/grafana: docs/sources/visualizations/panels-visualizations/visualizations/logs/index.md

_Logs_ are structured records of events or messages generated by a system or application&mdash;that is, a series of text records with status updates from your system or app. They generally include timestamps, messages, and context information like the severity of the logged event.

> The logs visualization displays these records from data sources that support logs, such as Elastic, Influx, and Loki. The logs visualization shows the log line body with expandable log details, displayed fields, and log-level indicators when those fields are available.

Reviewer: we need someone from @grafana/observability-logs to have a look at this I think?

Reviewer 2: I think the previous explanation made sense "The logs visualization shows, by default, the timestamp, a colored string representing the log status, the log line body, as well as collapsible log events that help you analyze the information generated." This is what users will see first before clicking on the log line which opens the log details to reveal indexed fields and structured metadata.

Author: Thanks for taking a look. We're trying out an automation that looks for drift between docs and code, and some the results are great, but some of them seem to go beyond drift. I'll revert this change.

Author: ```suggestion
The logs visualization displays these records from data sources that support logs, such as Elastic, Influx, and Loki. The logs visualization shows, by default, the timestamp, a colored string representing the log status, the log line body, as well as collapsible log events that help you analyze the information generated.
```

_Logs_ are structured records of events or messages generated by a system or application&mdash;that is, a series of text records with status updates from your system or app. They generally include timestamps, messages, and context information like the severity of the logged event.

The logs visualization displays these records from data sources that support logs, such as Elastic, Influx, and Loki. The logs visualization shows the log line body with expandable log details, displayed fields, and log-level indicators when those fields are available.

{{< figure src="/media/docs/grafana/panels-visualizations/screenshot-logs-v12.3.png" max-width="750px" alt="Logs visualization" >}}

When the log line details are open, you have access to the following additional options:

- Explain log line in Assistant
- Scroll to log line
- Show or hide log line
- Copy

> - Copy shortlink

Author: ```suggestion
- Copy to clipboard
- Copy shortlink
- Switch to condensed view
- Anchor to the right
```

- Pin or unpin the log line
- Show context
- Anchor to the right or display inline

> - Close log details sidebar or close details for this log

Author: ```suggestion
```

|      Option     |   Description   |
| --------------- | --------------- |
| Show timestamps | Show or hide the time column. This is the timestamp associated with the log line as reported from the data source. The default is off. |

> | Timestamp resolution | Choose the timestamp precision, such as milliseconds or nanoseconds. |

Author: ```suggestion
| Timestamp resolution | Choose the timestamp precision: **Milliseconds** or **Nanoseconds**. |
```

| Display log level | Show or hide the detected log level. |

> | Unique labels | Show or hide the unique labels column, which shows only non-common labels. |

Author: ```suggestion
| Font size | Choose from **Default** or **Small**. |
```

| Wrap lines | Turn line wrapping on or off. |
| Prettify JSON | Toggle the switch on to pretty print all JSON logs. This setting does not affect logs in any format other than JSON. |
| Enable columns for displayed fields | Show selected displayed fields as columns. |
| Enable logs highlighting | Use a predefined syntax coloring grammar to highlight relevant parts of the log lines. |
| Enable log details | Toggle the switch on to see an extendable area with log details including labels and detected fields. Each field or label has a stats icon to display ad-hoc statistics in relation to all displayed logs. The default setting is on. |
| Log Details panel mode | Choose to display the log details in a sidebar panel or inline, below the log line. |
| Enable infinite scrolling | Request more results by scrolling to the bottom of the logs list. |
| Show controls | Display controls to jump to the last or first log line and filter by log level. |
| Display download control | Display a control to download log lines when **Show controls** is enabled. |
| Font size | Select between the default font size and small font size. |

> | Deduplication | Hide log messages that are duplicates of others shown, according to your selected criteria. Choose from: <ul><li>**None** - Don't deduplicate log lines.</li><li>**Exact** - Ignoring ISO datetimes.</li><li>**Numbers** - Ignoring only those that differ by numbers such as IPs or latencies.</li><li>**Signature** - Removing successive lines with identical punctuation and white space.</li></ul> |

Author: ```suggestion
| Unique labels | Show or hide the unique labels column, which shows only non-common labels. |
| Deduplication | Hide log messages that are duplicates of others shown, according to your selected criteria. Choose from: <ul><li>**None** - Don't deduplicate log lines.</li><li>**Exact** - Ignoring ISO datetimes.</li><li>**Numbers** - Ignoring only those that differ by numbers such as IPs or latencies.</li><li>**Signature** - Removing successive lines with identical punctuation and white space.</li></ul> |
```
